gm4_disassemblers/data/gm4_disassemblers/functions/init.mcfunction

```diff
@@ -1,6 +1,7 @@
 scoreboard objectives add gm4_disassembler dummy
 scoreboard players set #100 gm4_disassembler 100
 scoreboard players set #2 gm4_disassembler 2
+scoreboard players add disassemble_diamonds gm4_disassembler 0
```

Comment:
I'd prefer a `gm4_disassemblers_config` scoreboard objective so that it's clear these are values that _should_ be modified by users. It'd also be nice to have a consistent standard for config scoreboards for future data packs (currently the only other data packs with a config is orbis, which uses `gm4_orbis_config` and the guidebook, which uses `gm4_guide_config`)

Reply:
I actually completely agree but I seem to remember from the meeting that someone (@Bloo-dev ? ) was against this. I think he wanted the impact of this setting to be as minimal as possible. Perhaps he can remind us why he wanted it this way?

Reply:
That is true, yes. I don't think an extra scoreboard is justified, even though the impact of one is minimal.